src/DynamoCore/Updates/UpdateManager.cs

```diff
@@ -164,6 +165,11 @@ public interface IUpdateManager
         /// Get the current name of the Host
         /// </summary>
         String HostName { get; set; }
+
+        /// <summary>
+        /// Analytics info of the host
+        /// </summary>
+        HostContextInfo HostInfo { get; set; }
```

Comment:
I am exposing this type defined in Analytics.Net here as we did for ProductInfo and CustomerInfo. The only difference is that `HostContextInfo` cant be constructed run time but rely on integrators to pass in, in that case, we will need this type in a couple of other places where I had to add reference.

Reply:
On another though.. This seems problematic in a way that integrators will now depend on Analytics.Net where this type is defined.. @mjkkirschner @pinzart This seems both API breaking and problematic.. Maybe we only expose two strings `ParentId` and `SessionId` and ask DynamoCore to construct them into `HostContextInfo`. What do you think?

Reply:
yeah, there seems to be 3 problems
* why is this property being added to IUpdateManager - what does this have to do with updates?
* this is breaking a public interface
* new dependency on analytics.net 


I'm fine with your proposed solution I think - not sure exactly what it would look like though - where would DynamoCore get those strings?

Reply:
@mjkkirschner @pinzart I decided to use the alternative approach on this. By inspecting https://git.autodesk.com/Alias/dynamo-plugin/blob/5324734fa78953c7575715733e59f1952b379d7b/src/DynamoIPC/UpdateManager.cs#L87, I think introducing this type at this level will not only break integration but also introduce extra dependency there

Reply:
@mjkkirschner These strings will be empty for Sandbox but passed in by integrators using CLI or when they define update manager. I think another good place might be `StartConfiguration` but given `HostName` is also in UpdateManager, for now I would like they all live in the same place